Alloy/commands/compile/index.js

```diff
@@ -706,7 +713,7 @@ function parseAlloyComponent(view, dir, manifest, noView, fileRestriction) {
 	}
 	var cCode = CU.loadController(files.CONTROLLER);
 	template.parentController = (cCode.parentControllerName !== '') ?
-		cCode.parentControllerName : "'BaseController'";
+    	cCode.parentControllerName : CU[CONST.DOCROOT_BASECONTROLLER_PROPERTY] || "'BaseController'";
```

Comment:
Yes, the project favors tabs over spaces.
Also, line 716 is a continuation of the previous line, so 1 tab should be added to the current indentation.